extensions/core/deployment/src/main/java/io/quarkiverse/cxf/deployment/QuarkusCxfProcessor.java

```diff
@@ -247,6 +247,8 @@ void buildXmlResources(BuildProducer<NativeImageResourceBuildItem> resources,
                         new GeneratedResourceBuildItem("META-INF/wsdl.plugin.xml", os.toByteArray()));
             }
 
+            System.clearProperty("javax.xml.accessExternalDTD");
```

Comment:
I think it would be "safer" to read the initial value before setting `"file, http"` and re-set that initial value instead of clearing the entire property?

Reply:
Good point @famod, will address this shortly!!